benchmarks/spectre_v1/README.md

```diff
@@ -0,0 +1,30 @@
+# Benchmarks of Spectre v1 Mitigations
+
+This folder has microbenchmarks to see the differences between the performance
+of Spectre v1 mitigations at a small scale.
+
+## Build instructions
+
+```
+$ bash
+$ cd safeside
+$ cmake -B build .
+$ make -C build
+
+# Everything should be built now.
+
+# Run the microbenchmark with no mitigation enabled.
+$ ./build/benchmarks/spectre_v1/unmitigated
+84733
+
+# If your compiler is Clang, then this is also built.
+# Run the microbenchmark with speculative load hardening enabled.
+./build/benchmarks/spectre_v1/slh_mitigated
+193599
+
+# If you want to explicitly set your compiler to clang.
+
+$ cmake -B build . \
+  -DCMAKE_CXX_COMPILER=path/to/your/clang++
```

Comment:
easier:
```
$ CC=clang CXX=clang cmake -B build .
```

On systems like Ubuntu 18.04 where Clang, even if installed, might be too old to have SLH:

```
$ sudo apt install clang-10
$ CC=clang-10 CXX=clang++-10 cmake -B build .
```